JS/fluentRevealNavbar.uc.js

```diff
@@ -16,16 +16,32 @@
       // if true, show the effect on bookmarks on the toolbar
       includeBookmarks: true,
 
+      // if ture, show the effect on the urlbar as well
+      includeUrlBar: true,
+
       // the color of the gradient. default is sort of a faint baby blue. you may prefer just white, e.g. hsla(0, 0%, 100%, 0.05)
-      lightColor: "hsla(224, 100%, 80%, 0.15)",
+      // lightColor: "hsla(224, 100%, 80%, 0.15)",
+
+      // the color of the gradient. default is the browser's color of navbar button's hover 
+      lightColor: "var(--button-hover-bgcolor)",
 
       // how wide the radial gradient is.
       gradientSize: 50,
 
       // whether to show an additional light burst when clicking an element. (not recommended)
       clickEffect: false,
+
+      // don't process mouse movements greater than {gradientSize}px from top of the screen, in order to reduce system load.
+      // disable if you modified the ui to have toolbar buttons on different side of the screen (left, right or bottom)
+      filterDy: true,
```

Comment:
What does filterDy mean? A little confusing maybe, I'm not familiar with this term.

Reply:
It meant something like a short form of "filter out dy coordinates too far from toolbar". Guess I couldn't come up with anything more clear. Maybe `filterOutDy` would be better?

Reply:
Oh, I see what you mean. No that's not particularly better haha. It's fine. If there was a more well known term we could use it, but it's a little hard to express the idea concisely, so I guess this is right.